public/static/app/perf-bb/PerfBBOutliersCtrl.js

```diff
@@ -627,7 +627,15 @@ mciModule.controller('PerfBBOutliersCtrl', function (
         }
         const mark = _.findWhere(marks, matcher);
         doc.marked = !!mark;
+        return doc;
       });
+
+      if (this.mode === 'muted')
```

Comment:
[nit] I'm a big fan of always using braces even if the block is a single line. I've seen too many cases where not including them causes bugs in the future.

Reply:
:+1:  done